components/secondary_button/README.md

```diff
@@ -0,0 +1,90 @@
+---
+description: Secondary buttons allow users to carry out a supporting action
+title: Secondary Button
+status: Draft
+author: orinevares
+---
+
+![Status](https://img.shields.io/badge/Recommended-Draft-orange.svg)
+> Last Updated: March 27, 2019
+
+# Secondary Button
+Secondary buttons allow users to carry out a supporting action on your service, such as Back or Cancel.
+
+## Visual Example
+
+<component-preview path="components/secondary_button/sample.html" height="100px" width="800px"> </component-preview>
+
+## Use This For
+* A supporting action you want users to take on your site.
+
+## Don't Use This For
+*	Primary actions on your page such as 'Submit', or 'Start'. Use a [Primary Button](https://developer.gov.bc.ca/Design-System/Primary-Button) instead.
+
+## Design Guidance
+*	Use for supporting actions or options you don't want users to default to
```

Comment:
I reviewed this line with Kaitlyn and we determined we can just get rid of it. This is information that's duplicated in the "use this for" and the "don't use this for"